setup.py

```diff
@@ -97,6 +98,26 @@ class CommonInstall:
     """Class with common method used by children classes."""
 
     @staticmethod
+    def add_jinja2_to_path():
+        """Add Jinja2 into sys.path.
+
+        As Jinja2 may be installed during this setup, it will not be available
+        on runtime to be used here. So, we need to look for it and append it on
+        the sys.path.
+        """
+        #: First we find the 'site_pkg' directory
+        site_pkg = None
+        for path in sys.path:
+            if PurePosixPath(path).stem == 'site-packages':
```

Comment:
I thought (because of the documentation) that Path didn't had the stem property. I've checked and it have, so I'm changing it.